feature-toggles/feature-toggles-impl/src/main/java/com/duckduckgo/feature/toggles/impl/MetricPixelInterceptor.kt

```diff
@@ -112,7 +112,7 @@ class MetricPixelInterceptor @Inject constructor(
         logcat { "Pixel URL request dropped: ${chain.request()}" }
 
         return Response.Builder()
-            .code(200)
+            .code(500)
```

Comment:
Not strong about this, I just find annoying that all the pixels we are dropping are marked as 200 so you see the pixel sent log. This is to avoid that.